less/_components/bootstrap/buttons.less

```diff
@@ -15,7 +15,12 @@
 
 /* Custom buttons */
 .btn {
-  text-transform: @btn-text-transform;
+  & {
+    text-transform: @btn-text-transform;
+    font-weight: 600;
```

Comment:
Let's override this variable for this:@btn-font-weight;
https://github.com/twbs/bootstrap/blob/v3-dev/less/buttons.less